client/jetpack-connect/utils.js

```diff
@@ -61,7 +61,7 @@ export const authQueryPropTypes = PropTypes.shape( {
 
 export function addCalypsoEnvQueryArg( url ) {
 	let calypsoEnv = config( 'env_id' );
-	if ( window && window.COMMIT_SHA && window.location.host === 'calypso.live' ) {
+	if ( window && window.COMMIT_SHA && isCalypsoLive() ) {
```

Comment:
The check for `window` will crash with ReferenceError if `window` indeed doesn't exist (e.g., in Node). But that's a preexisting bug that seems to not have caused any harm so far. Pushing a fix wouldn't be bad anyway.

Reply:
Thanks, fixed in 3e4c47c